kong/init.lua

```diff
@@ -190,6 +190,10 @@ local function execute_plugins_iterator(plugins_iterator, phase, ctx)
     kong_global.set_namespaced_log(kong, plugin.name)
     plugin.handler[phase](plugin.handler, configuration)
     kong_global.reset_log(kong)
+
+    if plugin.handler._go then
+      ngx.ctx.ran_go_plugin = true
```

Comment:
I think we should use the `ctx` argument here instead of directly referencing `ngx.ctx`.